Library/Homebrew/dev-cmd/bump.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+require "cli/parser"
+
+module Homebrew
+  module_function
+
+  def bump_args
+    Homebrew::CLI::Parser.new do
+      usage_banner <<~EOS
+        `bump` [<options>]
+
+        Display out-of-date brew formulae and the latest version available.
+        Also displays whether a pull request has been opened with the URL.
+      EOS
+      flag "--formula=",
+           description: "Return results for package by name."
+      flag "--limit=",
+           description: "Limit number of package results returned."
+      switch :verbose
+      switch :debug
+    end
+  end
+
+  def bump
+    args = bump_args.parse
+
+    requested_formula = args.formula
+    requested_limit = args.limit.to_i if args.limit.present?
+
+    raise FormulaUnavailableError, requested_formula if requested_formula && !validate_formula(requested_formula)
+
+    repology_data = if requested_formula
+      Repology.single_package_query(requested_formula)
+    else
+      Repology.parse_api_response
+    end
+
+    if repology_data.blank?
+      ohai "No Repology data found."
```

Comment:
I think it'd be cool to still display the formula, livecheck and PR data in this case.